tokio-postgres/src/config.rs

```diff
@@ -69,6 +69,45 @@ pub(crate) struct Inner {
     pub(crate) target_session_attrs: TargetSessionAttrs,
 }
 
+// Omit password from debug output
+impl fmt::Debug for Inner {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
```

Comment:
This should use https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct.

Reply:
Fixed. I added a struct to show an underscore with debug_struct, to prevent confusion from showing (). Let me know if you just want to use unit instead.